fenicsxprecice/fenicsxprecice.py

```diff
@@ -180,6 +180,38 @@ def read_data(self, mesh_name, read_data_name, dt):
 
         return copy.deepcopy(read_data)
 
+    def read_data_at_coordinates(self, mesh_name, read_data_name, coordinates, dt):
+        """
+        Read data from preCICE at specified coordinates. This function uses the JIT-mapping feature.
+
+        Args:
+            mesh_name: Specifies for which mesh the data shall be read
+            read_data_name: Specifies the data name of the mesh
+            coordinates (list): List of coordinates where preCICE reads the data
+            dt: offset within time window
+
+        Returns:
```

Comment:
```suggestion
        Returns
        -----------
```
Let us stick to one docstring style.